ReVampyr_coulomb.py

```diff
@@ -399,3 +401,19 @@
     print('GJmK_21_i', GJmK_21_i)
     print('GJmK_22_i', GJmK_22_i)
     print('E_C_G', E_tot_JK - GJmK_11_r - (2.0 *light_speed**2))
+
+if args.colgau == 'gauge':
```

Comment:
What is this part doing in the coulomb file?

Reply:
I have removed it is not necessary